x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_significant_events_view/add_significant_event_flyout/add_significant_event_flyout.tsx

```diff
@@ -174,12 +176,24 @@ export function AddSignificantEventFlyout({
           });
           telemetryClient.trackSignificantEventsSuggestionsGenerate({
             duration_ms: Date.now() - startTime,
+            count: numberOfGeneratedQueries,
```

Comment:
the number of invalid queries generated should be an interesting metric as well.
On a side note I'm wondering why we let the server return invalid queries and not validate there instead ?